WalletWasabi.Gui/Helpers/NotificationHelpers.cs

```diff
@@ -43,7 +43,7 @@ public static void Warning(string message, string title = "Warning!")
 				.Show(new Notification(title, message, NotificationType.Warning, TimeSpan.FromSeconds(7))));
 		}
 
-		public static void Error(string message, string title = "Error")
+		public static void Error(string message, string title = "Error!")
```

Comment:
In the rest of the changes where there is an error you replaced `!` by `.` and here, instead of ending the message with a `.` you introduced a `!`. Is this okay? What the rationale of this?

Reply:
Yes, because I want to unify it, and this is the **title** of the notification box, the other two titles are `Success!` and `Warning!`.